tests/environments/storage/test_docker_storage.py

```diff
@@ -515,6 +527,32 @@ def test_create_dockerfile_with_weird_flow_name():
             )
 
 
+def test_create_dockerfile_with_weird_flow_name_custom_prefect_dir():
+    with tempfile.TemporaryDirectory() as tempdir_outside:
```

Comment:
Instead of using `tempfile.TemporaryDirectory`, I recommend using pytest's `tmpdir` fixture: https://docs.pytest.org/en/stable/tmpdir.html

*(I know `tempfile.TemporaryDirectory()` is used elsewhere already, we're slowly migrating towards making better use of fixtures as tests are updated)*

Reply:
oh sure, I can do that. I just copied this from another test and added in the `prefect_directory` kwarg.

Reply:
Ok I just changed the two tests in this PR that were using `tempfile.TemporaryDirectory()` to `tmpdir`.

Thanks for showing me that! I think I'll start using it in other projects.